app/controllers/application_controller.rb

```diff
@@ -13,9 +13,15 @@ class ApplicationController < ActionController::API
 
   private
 
+  # This would get moved into the Insights Common optional_auth paths.
+  CATALOG_OPTIONAL_AUTH_PATHS = [
+    %r{\A/health\z},
```

Comment:
This would just be added here:
https://github.com/RedHatInsights/insights-api-common-rails/blob/6c308db7d77d16201442a7687bc761fd05d7dbe1/lib/insights/api/common/request.rb#L18-L21
to remove the extra method below.